ground/gcs/src/plugins/sysalarmsmessaging/sysalarmsmessagingplugin.cpp

```diff
@@ -89,42 +90,38 @@ bool SysAlarmsMessagingPlugin::initialize(const QStringList &arguments, QString
  */
 void SysAlarmsMessagingPlugin::updateAlarms(UAVObject* systemAlarm)
 {
-    Q_ASSERT(systemAlarm->getObjID() == SystemAlarms::OBJID);
```

Comment:
The proper safe way is to use DEPENDPATH in the qt project files so that header dependencies aren't ignored like they are currently.